docsys/src/main/java/io/nosqlbench/docsys/endpoints/DocsysMarkdownEndpoint.java

```diff
@@ -180,12 +180,6 @@ private void init(boolean reload) {
     }
 
     private void enable(Set<String> enabled) {
-
-        Set<String> toEnable = new HashSet<>();
-        if (this.enables !=null) {
-            toEnable.addAll(this.enables);
-        }
```

Comment:
unused and the enable method is always called with `this.enables` it seems

Reply:
This does look like a scoping bug. The behavior isn't active in any code so there is no impact. This can be re-added correctly if/when needed.